### PR TITLE
auto: improve log messages

### PR DESCRIPTION
**Pull request by hdonnay**
Adds some helpers and improves logging around missing cgroup interface files.

See-also: #2092

### CODE REVIEW

File: initialize/auto/cpu_linux.go [hunk 1]
```diff
@@ -43,9 +41,16 @@ func CPU() {
 }
 
 func cgLookup(r fs.FS) (int, error) {
+	usingDefault := func() { infoLog("no CPU quota set, using default") }
```
Comment (crozzy) [inline initialize/auto/cpu_linux.go]: Nit: What is the idea for not having these in-line? For me the pros of one string are outweighed by the cons of more indirection.

**Review by crozzy — DISMISSED** [review]
SGTM, one comment, feel free to merge as is

File: initialize/auto/cpu_linux.go (hunk 1 above)
Comment (hdonnay) [reply initialize/auto/cpu_linux.go]: yeah, I'll just rewrite this to use a constant string :+1:

**Review by crozzy — APPROVED** [review]
LGTM

**Pull request merged**